specification/cognitiveservices/data-plane/Language/preview/2022-07-01-preview/analyzetext.json

```diff
@@ -0,0 +1,2698 @@
+        }
+      ],
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "LanguageDetectionResults"
+    },
+    "LanguageDetectionResult": {
+      "type": "object",
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "Response by document",
+          "items": {
+            "$ref": "#/definitions/LanguageDetectionDocumentResult"
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/PreBuiltResult"
+        }
+      ],
+      "required": [
+        "documents"
+      ]
+    },
+    "LanguageDetectionDocumentResult": {
+      "type": "object",
+      "properties": {
+        "detectedLanguage": {
+          "description": "Detected Language.",
+          "$ref": "#/definitions/DetectedLanguage"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/DocumentResult"
+        }
+      ],
+      "required": [
+        "detectedLanguage"
+      ]
+    },
+    "DetectedLanguage": {
+      "type": "object",
+      "required": [
+        "name",
+        "iso6391Name",
+        "confidenceScore"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Long name of a detected language (e.g. English, French)."
+        },
+        "iso6391Name": {
+          "type": "string",
+          "description": "A two letter representation of the detected language according to the ISO 639-1 standard (e.g. en, fr)."
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
+          "description": "A confidence score between 0 and 1. Scores close to 1 indicate 100% certainty that the identified language is true."
+        }
+      }
+    },
+    "AnalyzeTextJobState": {
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/JobState"
+        },
+        {
+          "$ref": "#/definitions/TasksState"
+        },
+        {
+          "$ref": "#/definitions/AnalyzeTextJobStatistics"
+        }
+      ]
+    },
+    "Pagination": {
+      "properties": {
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "AnalyzeTextJobStatistics": {
+      "properties": {
+        "statistics": {
+          "$ref": "common.json#/definitions/RequestStatistics"
+        }
+      },
+      "type": "object"
+    },
+    "TasksState": {
+      "properties": {
+        "tasks": {
+          "properties": {
+            "completed": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "failed": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "inProgress": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "total": {
+              "type": "integer",
+              "format": "int32"
+            },
```

Comment:
Please remove the `format: int32` from these elements as they were not present in prior GA versions and adding them now could cause breaking changes in generated clients.

Reply:
1. Seems like the original version was off in case of the swagger, yes.
2. I can remove the format:int32 changes but that will causing linting issues (which is the primary reason they were added). I'm assuming we're ok with failing the format linting check. Will there be a suppression added in that case?

Reply:
You can suppress the linting issues. The link in each issue should describe how but, yes, effectively we could just ignore those.

@mikekistler does that feedback apply to adding `format: url` to strings as well?

Reply:
I think adding `format: uri` is okay if it can be done without breaking changes. If it gets rid of a transform for .NET and the other languages don't care, then I'd say it's a positive change.